tests/testthat/helper-vdiffr.R

```diff
@@ -0,0 +1,16 @@
+## There are currently issues installing vdiffr on CI for R version 3.2. We do
+## not run the visual tests on that version of R, but R CMD check will still
+## fail if the suggested package isn't available. Rather than ignore that check
+## completely, this more specific helper will raise an error if we're meant to
+## be running vdiffr tests but vdiffr is not installed; otherwise if vdiffr is
+## not available then it will skip.
+
+if (requireNamespace("vdiffr", quietly = TRUE)) {
+  expect_doppelganger <- vdiffr::expect_doppelganger
+} else {
+  # If vdiffr is not available and visual tests are not explicitly disabled, raise error.
+  if (!identical(Sys.getenv("VDIFFR_RUN_TESTS"), "false")) {
+    rlang::abort("vdiffr is not installed")
```

Comment:
I am a little confused because I cannot load the package from a clone of the repo with `devtools::load_all()`. It won't let it load without vdiffr installed (getting error from this abort call). Since this is a suggested, not required, package, I am not sure that's the behavior you want it to have.

Reply:
The idea here is that if you are going to be testing the package including running the visual tests, you need to have vdiffr installed. So I think this is expected, but it may be too heavy-handed. I can remove this (but running the tests will still error if vdiffr is not installed).

Reply:
I see. That is a tough call, then. There should be no real need to load the package from a local clone unless someone is trying to do something more with the package itself. In that case, being heavy-handed isn't a big deal since testing is a good thing when extending a package. At the same time, if someone doesn't have testthat or mockery installed, the tests wouldn't be able to run either, but they could still load the package because that isn't being checked for in the same way that vdiffr is. 🤷‍♀️ I'm not sure what the ideal solution is.

Interestingly, 5 tests fail on VM (linux). I would expect the 5th test to fail because it's comparing images, but the first 4 appear to be from a "Missing or unexported object: 'ggplot2::binned_scale'". That's not a part of this PR, though.
![image](https://user-images.githubusercontent.com/25048342/93646764-b6d78680-f9bb-11ea-87ef-4dab7b2265a3.png)

Overall, seems like the stuff related to this PR works as intended!

Reply:
I'm guessing your VM has an older version of ggplot2 that doesn't support binned scales (they're a relatively new feature). Do you see this warning if you try to used the binned scales on your VM?
https://github.com/Sage-Bionetworks/sagethemes/blob/9644d00604e2d43fe40e7bfc7392c108fe4b6528/R/scale-colour-sage-b.R#L33

Feel free to open a separate issue for this, we may want to make the test expectations conditional on the ggplot2 version.

Reply:
I do see that warning. I haven't updated ggplot2 in a while. I can open an issue for that.

Reply:
Sounds good. I think I've addressed everything you raised that's specific to this PR; if so, will you approve?